### PR TITLE
Add tutorial for perception setup for A1.

### PR DESCRIPTION
**Pull request by yxyang**
see title.

### CODE REVIEW

File: Tutorials/a1_perception_setup.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+title: Set Up Perception Pipeline for A1
+parent: Tutorials
+nav_order: 99
```
Comment (romesco) [inline Tutorials/a1_perception_setup.md]: The nav order should be something other than the template one! (so like 2 or 3 for example)

File: Tutorials/a1_perception_setup.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+# Setting Up Perception Pipeline for A1
+This tutorial goes through steps to set up the perception pipeline for the A1 quadruped robot, which includes installing drivers for the RealSense D435i camera, installing `pytorch` and `torchvision` specifically for the TX2 platform.
+
+## Connecting to the Robot
+By default, the A1 robot has a built-in wifi HotSpot named `UnitreeRoboticsA1-XXX`, where `XXX` is the robot serial number. To access the onboard TX2, connect to the wifi hotspot (default password is `00000000`). Then you can `ssh` into the robot by running:
+
+`ssh unitree@192.168.123.12`
+
+with password `123`.
```
Comment (romesco) [inline Tutorials/a1_perception_setup.md:18]: I would caution against putting 'non-default' password here! Perhaps set a 'standard' password that is known internally, but not shared on the internet.

File: Tutorials/a1_perception_setup.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  app.run(main)
+```
+
+## Setup `pytorch` 
+
+TX2 has specific hardware / cuda versions that standard `pip`-installed wheels do not support. The official [thread](https://forums.developer.nvidia.com/t/pytorch-for-jetson-version-1-9-0-now-available/72048) contains a set of pre-built wheels for pytorch and jetson hardware.
+
+For my A1 robot, the cuda version installed only supports `pytorch` up to `1.4.0`. To install the specific version, run:
+
+`wget https://nvidia.box.com/shared/static/ncgzus5o23uck9i5oth2n8n06k340l6k.whl -O torch-1.4.0-cp36-cp36m-linux_aarch64.whl`
```
Comment (romesco) [inline Tutorials/a1_perception_setup.md:89]: This is great. I think for tutorials, it is ideal to give barebones commands to get the reader where they need to be as quickly as possible. It is always annoying to me when someone provides a link which I then have to follow and dig through a separate webpage.

File: Tutorials/a1_perception_setup.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+and install it by running:
+
+`pip3 install Cython numpy torch-1.4.0-cp36-cp36m-linux_aarch64.whl`
+
+Note: do not install `numpy` version `1.19.5` yet as it is not supported.
```
Comment (romesco) [inline Tutorials/a1_perception_setup.md:95]: isnt the default `numpy` on PyPI version `1.21.1` which is a later release?

**Review by romesco — APPROVED** [review]
Take a look at my suggestions! Otherwise I think its really great! I am happy you took the time to write it up - it will likely save others a lot of suffering int he future XD

File: Tutorials/a1_perception_setup.md (hunk 2 above)
Comment (yxyang) [reply Tutorials/a1_perception_setup.md:18]: These are default passwords from Unitree which are the same for all robots, so there isn't much issue in sharing it.

File: Tutorials/a1_perception_setup.md (hunk 4 above)
Comment (yxyang) [reply Tutorials/a1_perception_setup.md:95]: Not for aarch64 it seems...

File: Tutorials/a1_perception_setup.md (hunk 1 above)
Comment (yxyang) [reply Tutorials/a1_perception_setup.md]: Done.

**Pull request merged**